packages/selinux-policy/rules.cil

```diff
@@ -166,10 +166,28 @@
 ; Subjects that must run verified code can execute immutable objects, since
 ; those are all protected by dm-verity.
 (allow verified_s immutable_o (files (execute)))
+(allow kernel_t immutable_o (files (execute)))
 
 ; Subjects that must run verified code cannot execute mutable objects.
 (neverallow verified_s mutable_o (files (execute)))
 
+; Ideally the kernel would also be denied permission to execute mutable
+; objects. However, this breaks certain scenarios such as serving files
+; over NFS, where the kernel's permissions are checked.
+(allow kernel_t mutable_o (file (execute)))
```

Comment:
This is the actual fix to the NFS serving issue.

Note that `files (execute)` grants both "execute" and "execute_no_trans"; I've moved to the more specific `file` here in order to split up those two permissions.